js/main.js

```diff
@@ -0,0 +1,127 @@
+$(document).ready(function () {
+
+    $("#loginForm").submit(function (event) {
+
+        var email = $("#email").val();
+        var password = $("#pwd").val();
+
+        $.ajax({
+            type: "POST",
+            data: {
+                action: 'authenticateUser',
+                table: 'test_table1',
+                email: email,
+                password: password
+            },
+            url: "ajax_php/actions.php",
+            beforeSend: function () {
+                console.log('sending request ...');
+            },
+            dataType: "html",
+            async: true,
+            success: function (data) {
+
+                var result = JSON.parse(data);
+                const token = result['token'];
+
+                if (result['error']) {
+
+                    trackLogin(false);
+
+                    $('#email').val('');
+                    $('#pwd').val('');
```

Comment:
chiar daca este html / css, incearca sa nu folosesti prescurtari in id-urile elementelor. le faci foarte greu de gasit.